FaceRecognizer.py

```diff
@@ -89,10 +91,15 @@ def shape_to_np(shape, dtype="int"):
     (x, y, w, h) = face_utils.rect_to_bb(rect)
     faceOrig = imutils.resize(image[y:y + h, x:x + w], width=256)
     faceAligned = face.align(image, gray, rect)
-    cv2.rectangle(image, (x, y), (x + w, y + h), (0, 255, 0), 2)
 
+    gray= cv2.cvtColor(faceAligned, cv2.COLOR_BGR2GRAY)

```

Comment:
FaceAligned Img had an empty space.
I changed it to gray to find the coordinates of the empty space.